css/style.css

```diff
@@ -360,12 +360,17 @@ nav.navbar.navbar-default {
 	margin: 2em 0;
 }
 
-.entry-meta {
-	margin: 1em 0;
-	font-size: 14px;
+.article-head p.entry-meta,
+.non-link a {
+	font-size: 12px;
 	color: #777;
 }
 
+.non-link a {
```

Comment:
I think I got it to spit out non-linking display of categories...

On Thu, Jun 30, 2016 at 3:14 PM frances botsford <notifications@github.com>
wrote:

> In css/style.css
> <https://github.com/MITLibraries/MITLibraries-child/pull/68#discussion_r69193180>
> :
>
> >  	color: #777;
> >  }
> >
> > +.non-link a {
>
> is there a way to just not have it spit out a link?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MITLibraries/MITLibraries-child/pull/68/files/9a09164b451403c9a736380c943a7e6c8279e8e1#r69193180>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABv1VXbpTgVFocJujoTNgBZRz5EiqBmdks5qRBWugaJpZM4JBl1F>
> .
>